RevenueCatUI/Views/LoadingPaywallView.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  LoadingPaywallView.swift
+//  
+//
+//  Created by Nacho Soto on 7/21/23.
+//
+
+import RevenueCat
+import SwiftUI
+
+/// A `PaywallView` suitable to be displayed as a loading placeholder.
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+struct LoadingPaywallView: View {
+
+    var body: some View {
+        LoadedOfferingPaywallView(
+            offering: .init(
+                identifier: Self.offeringIdentifier,
+                serverDescription: "",
+                metadata: [:],
+                paywall: .default,
+                availablePackages: [
+                    Self.monthlyPackage,
+                    Self.weeklyPackage,
+                    Self.annualPackage
+                ]
+            ),
+            paywall: .default,
+            mode: .fullScreen,
+            introEligibility: Self.introEligibility,
+            purchaseHandler: Self.purchaseHandler
+        )
+        .disabled(true)
+        .redacted(reason: .placeholder)
+    }
+
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+private extension LoadingPaywallView {
+
+    static let introEligibility: TrialOrIntroEligibilityChecker = .init(checker: { packages in
+        return Dictionary(
+            uniqueKeysWithValues: packages.map { ($0, .unknown) }
+        )
+    })
+    static let purchaseHandler: PurchaseHandler = .init(
+        purchase: { _ in
+            fatalError("Should not be able to purchase")
+        },
+        restorePurchases: {
+            fatalError("Should not be able to purchase")
+        }
```

Comment:
Important to create these so we don't attempt to purchase fake packages.
The view is marked as `.disabled()` to make this impossible.